src/style.css

```diff
@@ -0,0 +1,112 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color: #3d4e61;
+  --border-color: #e1e7ed;
+}
+
+html {
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+.form-big {
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid var(--border-color);
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)